arcgis-ios-sdk-samples/Utility Network/Display utility association/DisplayUtilityAssociation.storyboard

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="15705" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES" initialViewController="wEJ-aH-5mo">
+    <device id="retina6_1" orientation="portrait" appearance="light"/>
+    <dependencies>
+        <deployment identifier="iOS"/>
+        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="15706"/>
+        <capability name="Safe area layout guides" minToolsVersion="9.0"/>
+        <capability name="documents saved in the Xcode 8 format" minToolsVersion="8.0"/>
+    </dependencies>
+    <scenes>
+        <!--Display Utility Association View Controller-->
+        <scene sceneID="nCJ-Ec-jTB">
+            <objects>
+                <viewController id="wEJ-aH-5mo" userLabel="Display Utility Association View Controller" customClass="DisplayUtilityAssociationVC" customModule="ArcGIS_Runtime_SDK_Samples" customModuleProvider="target" sceneMemberID="viewController">
+                    <view key="view" contentMode="scaleToFill" id="WcN-ei-ZB2">
+                        <rect key="frame" x="0.0" y="0.0" width="414" height="896"/>
+                        <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+                        <subviews>
+                            <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="og2-5E-h2J" userLabel="Map View" customClass="AGSMapView">
+                                <rect key="frame" x="0.0" y="20" width="414" height="793"/>
+                            </view>
+                            <toolbar opaque="NO" clearsContextBeforeDrawing="NO" contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="96r-bO-QXb">
```

Comment:
Please disable user interaction on the toolbar so that the bar button items cannot be tapped.

![Screen Shot 2020-05-05 at 3 52 50 PM](https://user-images.githubusercontent.com/2257493/81123675-87325100-8ee8-11ea-8e17-ab68ce021224.png)